alpha/declcfg/load_benchmark_test.go

```diff
@@ -0,0 +1,313 @@
+package declcfg_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/fs"
+	"math/rand"
+	"os"
+	"syscall"
+	"testing"
+	"testing/fstest"
+	"time"
+
+	"github.com/blang/semver/v4"
+	"github.com/operator-framework/api/pkg/lib/version"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/property"
+)
+
+func BenchmarkLoadFS(b *testing.B) {
+	fbc := generateFBC(b, 300, 450, 3000)
+	fsys := &memFS{MapFS: fstest.MapFS{}}
```

Comment:
Filesystem interactions are likely important in loading, so this benchmark is IMHO less useful than it could be if it just used a temporary directory and real files.

Reply:
And ideally it's not the same filesystem for each benchmark since they'll be memory-mapped if it is :)